aiconfig-docs/core/tabConstants.js

```diff
@@ -29,8 +29,8 @@ const pythonPackageManagers = [
 const defaultPythonPackageManager = "pip";
 
 const aiConfigLanguages = [
```

Comment:
Note the ordering here is actually what defines the tab ordering. I've just updated the tab ordering in code for clarity